dagmc_stats.py

```diff
@@ -300,3 +300,184 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
         coarseness.append(len(surf_area)/sum(surf_area))
 
     return coarseness
+
+
+def get_beta_angles(my_core, pair_tris, common_vert, beta_angles):
+    """
+    Gets the beta angles of given triangle pairs
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    pair_tris : triangle pair
+    common_vert : common vertices in the triangle pair
+    beta_angles: list that stores the beta angles of given triangle pair
+    """
+    vert1 = common_vert[0]
+    vert2 = common_vert[1]
+    side_lengths = []
+    
+    for m in range(2):
+        adj_verts = my_core.get_adjacencies(pair_tris[m], 0, op_type=1)
+        for n in range(3):
+            if adj_verts[n] != vert1 and adj_verts[n] != vert2:
+                other_vert = adj_verts[n]        
+        side_lengths.append(np.linalg.norm(my_core.get_coords(vert1)-my_core.get_coords(vert2)))
+        side_lengths.append(np.linalg.norm(my_core.get_coords(vert1)-my_core.get_coords(other_vert)))
+        side_lengths.append(np.linalg.norm(my_core.get_coords(vert2)-my_core.get_coords(other_vert)))
+        beta_angles.append(np.arccos((side_lengths[1] * side_lengths[1]
+                            + side_lengths[2] * side_lengths[2]
+                            - side_lengths[0] * side_lengths[0])
+                            /(2.0 * side_lengths[1] * side_lengths[2])))
+    
+
+def get_angles(my_core, tri, vert = None):
+    """
+    Gets the angles of the given triangle
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    tri: triangle
+    vert: the vertex which generates the triangle by get_adj
+
+    outputs
+    -------
+    angles : (list) the angles in the tri
+    """
+    
+    angles = []
+    side_lengths = []
+    coord_list = []
+    index = 3
+    if vert != None:
+        verts = my_core.get_adjacencies(tri, 0, op_type=1)
+        for i in range(len(verts)):
+            if verts[i] == vert:
+                index = i
+        for vert in verts:
+            coords = my_core.get_coords(vert)
```

Comment:
Is it possible to call `get_coords` with a list of vertices and get back all the coordinates at once?